README.md

```diff
@@ -208,15 +208,53 @@ If you are still uncertain you can use a broad default parameters that will filt
 
 ```r
 quality_report(folder_sequences = directory_sequences,
-               outputfile = "QC_report.html",
-               output_dir = "~/Desktop/project_folder/results_output",
-               folder_path_fcs = directory_flowdata,
-               probe1 = "my_probe_channel_1", probe2 = "Alexa.Fluor.700.A",
-               posvalue_probe1 = 600, posvalue_probe2 = 400,
-               raw_length = 0, trim_finish = 50, 
-               cdr3_start = 0, cdr3_end = 0)
+              outputfile = "QC_report.html",
+              output_dir = "~/Desktop/project_folder/results_output",
+              raw_length = 0, trim_finish = 50,
+              cdr3_start = 0, cdr3_end = 0)
+```
+
+Default for flow cytometry data:
+
+Flow cytometry data should be examined to determine the optimal parameters based on your dataset and experiment. It is important to check your flow data to see how the data is being processed, if it is already compensated, and if the cells were probed. These conditions affect which thresholds you should use.
+
+In order to see the available channels and thresholds use the function `fcs_processing()` to create an object `fcs_data`, after which you can see the available channels using: 
+
+```{r channels}
+#   colnames(fcs_data)
+
+```
+
+The `specificity` column is based on your selected channels and their thresholds and uses these thresholds to name your sorted cells. Probed samples  would have a `Pre.F` single-positive, `Post.F` single-positive, double-positive cells named as`DP`, and double-negative cells named as `DN`. Which can be seen using: 
```

Comment:
could you add this?
"
The value used as input to `posvalue_probe1` and `posvalue_probe1` is the Mean Fluorescence Intensity (MFI) for a given channel. 
"

Reply:
Addressed in e7848a3f4b8050d7acd6b7aa6d60980a68f65fb1